# Solaris-specific patch. alloca(3c) is not visible without
# including <alloca.h>.
# Not for upstream.
--- lib/glob/glob.c	2009-11-14 15:39:30.000000000 -0800
+++ lib/glob/glob.c	2010-09-01 10:28:10.161379481 -0700
@@ -34,6 +34,10 @@
 #  include <unistd.h>
 #endif
 
+#if defined (HAVE_ALLOCA) && defined (HAVE_ALLOCA_H)
+#  include <alloca.h>
+#endif
+
 #include "bashansi.h"
 #include "posixdir.h"
 #include "posixstat.h"
